components/transloadit/transloadit.app.mjs

```diff
@@ -1,11 +1,101 @@
+import Transloadit from "transloadit";
+import { LIMIT } from "../../common/constants.mjs";
```

Comment:
The constants.mjs file is currently outside of the component folder (/components/transloadit). Need to move it inside (/components/transloadit/common/).
```suggestion
import { LIMIT } from "./common/constants.mjs";
```